sci-mathematics/topcom/topcom-0.17.8-r1.ebuild

```diff
@@ -24,7 +24,6 @@ RDEPEND="${DEPEND}"
 BDEPEND="app-shells/tcsh"
 
 PATCHES=( "${FILESDIR}/${P}-buildsystem.patch" )
-HTML_DOCS=( "${DISTDIR}/TOPCOM-manual.html" )
```

Comment:
use doc && HTML_DOCS+=( "${DISTDIR}/TOPCOM-manual.html" )

Instead of 3-liner in src-install()

Reply:
I am not against that per see but I will say I am uncomfortable with using the `use` construct in global scope. It may just be silly from me. That being said I could put the line as is in `src_install` where I have the block. As to block versus one-liner I am also on the fence on the clarity of the style.